phpunit.xml.dist

```diff
@@ -1,19 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         colors="true"
+         bootstrap="vendor/autoload.php"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.1/phpunit.xsd"
+         cacheDirectory=".phpunit.cache">
+  <coverage/>
 
-<phpunit colors="true" bootstrap="vendor/autoload.php">
-    <testsuites>
-        <testsuite name="Behat Mink test suite">
-            <directory>tests</directory>
-        </testsuite>
-    </testsuites>
+  <testsuites>
+    <testsuite name="Behat Mink test suite">
+      <directory>tests</directory>
+      <exclude>./tests/Element/ElementTest.php</exclude>
+      <exclude>./tests/Selector/NamedSelectorTest.php</exclude>
```

Comment:
excluding some tests is a no-go. Our testsuite must run.

If those are not actually tests but abstract test cases, the right fix is to rename them rather than having to maintain exclude rules.

Reply:
These are abstract Tests. I changed the tests. There are of course multiple solutions. Please check if my solution is ok.